python/lsst/ts/m2com/mock/mock_server.py

```diff
@@ -280,9 +284,14 @@ async def _send_welcome_message(self) -> None:
         await self._message_event.write_hardpoint_list(hardpoints)
 
         await self._message_event.write_interlock(False)
-        await self._message_event.write_inclination_telemetry_source(
-            self.model.inclination_source
-        )
+
+        # Workaround of the mypy checking
```

Comment:
Is this comment in the right line? shouldn't it be in the line below? Also, you might have to do this because you are not annotating `control_parameters`. Although, since the dictionary can also host a float you will probably have to do this check here as well.. You could potentially get around it by doing something like:

```
is_external_source = self.model.control_parameters[
             "use_external_elevation_angle"
         ] is True
```

e.g. comparing the value of `is_external_source = self.model.control_parameters["use_external_elevation_angle"]` with `True`.

Reply:
Fixed in the commit: [d7beec9](https://github.com/lsst-ts/ts_m2com/pull/38/commits/d7beec9180505af36e222c539032fea7eadfb7e5). Thanks!